Mathlib/Topology/Homeomorph.lean

```diff
@@ -512,6 +512,77 @@ section
 
 variable (X Y Z)
 
+/-- `X ⊕ Y` is homeomorphic to `Y ⊕ X`. -/
+def sumComm : X ⊕ Y ≃ₜ Y ⊕ X where
+  toEquiv := Equiv.sumComm X Y
+  continuous_toFun := continuous_sum_swap
+  continuous_invFun := continuous_sum_swap
```

Comment:
This and the rest of the lemmas here seem to be dumped in the middle of a `Prod` section. There are a few lemmas here about `Sum` (e.g., `sumCongr`). Maybe this fits better there?

Reply:
`sumCongr` is right next to `prodCongr` and a `Congr` definition...
I moved this into its own section, right before the corresponding `Prod` section.